bundle/mvn/default-bundle/src/test/resources/application.properties

```diff
@@ -2,3 +2,7 @@
 server.port=9898
 zeebe.broker.gateway-address=localhost:26500
 zeebe.client.security.plaintext=true
+
+management.server.port=9080
+management.context-path=/actuator
+management.endpoints.web.exposure.include=metrics,health,prometheus
```

Comment:
Should we also apply these changes to `saas-bundle/src/main/resources/applicaiton.properties`? Otherwise it will only work for the local runtime, I think.

Reply:
Awesome catch ⭐

Reply:
It's actually almost identical already, just missing the `prometheus` endpoint. Not sure if it's important.

Reply:
It is, yes, otherwise that endpoint wouldn't be available

Reply:
It is - I add it there - thx!